.github/workflows/ci_cron_weekly.yml

```diff
@@ -0,0 +1,75 @@
+# GitHub Actions workflow that runs on a cron schedule.
+
+name: Cron Scheduled CI Tests
+
+on:
+  schedule:
+    # run at 9am UTC on Mondays
+    - cron: '0 9 * * 1'
+
+jobs:
+  # The linkcheck job tests that the links in the docs point to real places
+  # The if statement is to prevent cron from running on forks.
+  linkcheck:
+    runs-on: ubuntu-latest
+    if: github.repository == 'spacetelescope/jdaviz'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python to build docs with sphinx
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install base dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install tox
+    - name: Check links in docs using tox
+      run: |
+        tox -e linkcheck
+  # everything except the matrix is copied-and-pasted from ci_workflows.yml ... not ideal, maybe there's some way to do this by reference?
```

Comment:
Actions not very good with "by reference" part yet.

But if your goal is to run the same jobs in both push and cron, it is easier to put `on: schedule:` to your `ci_workflows.yml` instead of copying it here. Then for the job there you don't want to run in cron, you can use `if: ${{ github.event_name != 'schedule' }}`. Does that make sense?

Reply:
Hmm.  I had been intending to run a completely disjoint set of jobs - i.e. those at push are *not* supposed to run on the cron schedule.  But now that you mention it, maybe this is better?  I can see it either way - does anyone else have an opinion here?

(For those who might not be in on the lingo here, the "cron" are the weekly ones)